{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2006-4980",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [
          {
            "vendor_name": "python_software_foundation",
            "product": {
              "product_data": [
                {
                  "product_name": "python",
                  "version": {
                    "version_data": [
                      {
                        "version_value": "2.3"
                      },
                      {
                        "version_value": "2.3.1"
                      },
                      {
                        "version_value": "2.3.2"
                      },
                      {
                        "version_value": "2.3.3"
                      },
                      {
                        "version_value": "2.3.4"
                      },
                      {
                        "version_value": "2.3.5"
                      },
                      {
                        "version_value": "2.4"
                      },
                      {
                        "version_value": "2.4.2"
                      },
                      {
                        "version_value": "2.4.3"
                      },
                      {
                        "version_value": "2.5_alpha_1"
                      },
                      {
                        "version_value": "2.5_alpha_2"
                      },
                      {
                        "version_value": "2.5_beta_1"
                      },
                      {
                        "version_value": "2.5_beta_2"
                      },
                      {
                        "version_value": "2.5_beta_3"
                      },
                      {
                        "version_value": "2.5_final"
                      },
                      {
                        "version_value": "2.5_release_candidate_1"
                      },
                      {
                        "version_value": "2.5_release_candidate_2"
                      }
                    ]
                  }
                }
              ]
            }
          }
        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "NVD-CWE-Other"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "ftp://patches.sgi.com/support/free/security/advisories/20061001-01-P.asc"
        },
        {
          "url": "http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=391589"
        },
        {
          "url": "http://bugzilla.redhat.com/bugzilla/show_bug.cgi?id=208162"
        },
        {
          "url": "http://kb.vmware.com/KanisaPlatform/Publishing/882/5120103_f.SAL_Public.html"
        },
        {
          "url": "http://secunia.com/advisories/22276"
        },
        {
          "url": "http://secunia.com/advisories/22297"
        },
        {
          "url": "http://secunia.com/advisories/22303"
        },
        {
          "url": "http://secunia.com/advisories/22357"
        },
        {
          "url": "http://secunia.com/advisories/22358"
        },
        {
          "url": "http://secunia.com/advisories/22379"
        },
        {
          "url": "http://secunia.com/advisories/22448"
        },
        {
          "url": "http://secunia.com/advisories/22487"
        },
        {
          "url": "http://secunia.com/advisories/22512"
        },
        {
          "url": "http://secunia.com/advisories/22531"
        },
        {
          "url": "http://secunia.com/advisories/22639"
        },
        {
          "url": "http://secunia.com/advisories/23680"
        },
        {
          "url": "http://secunia.com/advisories/31492"
        },
        {
          "url": "http://security.gentoo.org/glsa/glsa-200610-07.xml"
        },
        {
          "url": "http://securitytracker.com/id?1017019"
        },
        {
          "url": "http://sourceforge.net/tracker/index.php?func=detail&aid=1541585&group_id=5470&atid=305470"
        },
        {
          "url": "http://support.avaya.com/elmodocs2/security/ASA-2006-229.htm"
        },
        {
          "url": "http://www.debian.org/security/2006/dsa-1197"
        },
        {
          "url": "http://www.debian.org/security/2006/dsa-1198"
        },
        {
          "url": "http://www.mandriva.com/security/advisories?name=MDKSA-2006:181"
        },
        {
          "url": "http://www.novell.com/linux/security/advisories/2006_25_sr.html"
        },
        {
          "url": "http://www.redhat.com/support/errata/RHSA-2006-0713.html"
        },
        {
          "url": "http://www.redhat.com/support/errata/RHSA-2008-0629.html"
        },
        {
          "url": "http://www.securityfocus.com/archive/1/archive/1/448244/100/100/threaded"
        },
        {
          "url": "http://www.securityfocus.com/archive/1/archive/1/456546/100/200/threaded"
        },
        {
          "url": "http://www.securityfocus.com/bid/20376"
        },
        {
          "url": "http://www.ubuntu.com/usn/usn-359-1"
        },
        {
          "url": "http://www.vmware.com/support/vi3/doc/esx-3069097-patch.html"
        },
        {
          "url": "http://www.vmware.com/support/vi3/doc/esx-9986131-patch.html"
        },
        {
          "url": "http://www.vupen.com/english/advisories/2006/3940"
        },
        {
          "url": "http://www.vupen.com/english/advisories/2006/5131"
        },
        {
          "url": "http://zoehep.xent.com/~bsittler/python2.4-2.4.3_unicodeobject.c.diff"
        },
        {
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/29408"
        },
        {
          "url": "https://issues.rpath.com/browse/RPL-702"
        },
        {
          "url": "https://launchpad.net/distros/ubuntu/+source/python2.4/+bug/56633"
        },
        {
          "url": "https://oval.cisecurity.org/repository/search/definition/oval%3Aorg.mitre.oval%3Adef%3A10789"
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "Buffer overflow in the repr function in Python 2.3 through 2.6 before 20060822 allows context-dependent attackers to cause a denial of service and possibly execute arbitrary code via crafted wide character UTF-32/UCS-4 strings to certain scripts."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "operator": "OR",
        "cpe": [
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.3",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.3:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.3.1",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.3.1:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.3.2",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.3.2:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.3.3",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.3.3:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.3.4",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.3.4:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.3.5",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.3.5:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.4",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.4:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.4.2",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.4.2:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.4.3",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.4.3:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.5_alpha_1",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.5_alpha_1:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.5_alpha_2",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.5_alpha_2:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.5_beta_1",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.5_beta_1:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.5_beta_2",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.5_beta_2:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.5_beta_3",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.5_beta_3:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.5_final",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.5_final:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.5_release_candidate_1",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.5_release_candidate_1:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:python_software_foundation:python:2.5_release_candidate_2",
            "cpe23Uri": "cpe:2.3:a:python_software_foundation:python:2.5_release_candidate_2:*:*:*:*:*:*:*"
          }
        ]
      }
    ]
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "version": "2.0",
        "vectorString": "(AV:N/AC:L/Au:N/C:P/I:P/A:P)",
        "accessVector": "NETWORK",
        "accessComplexity": "LOW",
        "authentication": "NONE",
        "confidentialityImpact": "PARTIAL",
        "integrityImpact": "PARTIAL",
        "availabilityImpact": "PARTIAL",
        "baseScore": 7.5
      },
      "severity": "HIGH",
      "exploitabilityScore": 10.0,
      "impactScore": 6.4,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": true,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": false
    }
  },
  "publishedDate": "2006-10-10T04:06Z",
  "lastModifiedDate": "2017-10-11T01:31Z"
}